### PR TITLE
Refresh Helm chart

### PR DESCRIPTION
**Pull request by wtty-fool**
Towards https://github.com/giantswarm/giantswarm/issues/11215

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: helm/rbac-operator/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+app.kubernetes.io/name: {{ include "name" . | quote }}
+app.kubernetes.io/instance: {{ .Release.Name | quote }}
+{{- end -}}
+
+{{/*
+RBAC labels
+*/}}
+{{- define "labels.rbac" -}}
+{{ include "labels.common" . }}
+giantswarm.io/service-type: "managed"
```
Comment (wtty-fool) [inline helm/rbac-operator/templates/_helpers.tpl:42]: This is new; LMK if it makes sense.

Comment (kopiczko) [reply helm/rbac-operator/templates/_helpers.tpl:42]: Where does that come from?

File: helm/rbac-operator/templates/_resource.tpl [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+{{- define "resource.psp.name" -}}
+{{- include "resource.default.name" . -}}-psp
+{{- end -}}
+
+{{- define "resource.pullSecret.name" -}}
+{{- include "resource.default.name" . -}}-pull-secret
+{{- end -}}
+
+{{- define "resource.default.namespace" -}}
+giantswarm
```
Comment (kopiczko) [inline helm/rbac-operator/templates/_resource.tpl:27]:
```suggestion
{{ .Release.Namespace }}
```

File: helm/rbac-operator/templates/rbac-tenant-admin.yaml [hunk 3]
```diff
@@ -4,8 +4,7 @@ kind: ClusterRole
 metadata:
   name: tenant-admin
   labels:
-    app: rbac-operator
-    giantswarm.io/service-type: "managed"
```
Comment (kopiczko) [inline helm/rbac-operator/templates/rbac-tenant-admin.yaml:8]: I see. I'd leave `_helpers.tpl` untouched as it's copy/pasteable at this point and leave this label as an extra one.

@corest do we need this label at all?

File: helm/rbac-operator/templates/_helpers.tpl (hunk 1 above)
Comment (kopiczko) [reply helm/rbac-operator/templates/_helpers.tpl:42]: https://github.com/giantswarm/rbac-operator/pull/38#discussion_r435814803

File: helm/rbac-operator/templates/rbac-tenant-admin.yaml (hunk 3 above)
Comment (wtty-fool) [reply helm/rbac-operator/templates/rbac-tenant-admin.yaml:8]: No response, so I decided to roll back to common `_helpers.tpl`.

Comment (kopiczko) [reply helm/rbac-operator/templates/rbac-tenant-admin.yaml:8]: You have my blessing :D

**Pull request merged**